dist/obs/OSRT:OriginUpdateInitialBlacklist.xml

```diff
@@ -0,0 +1,5 @@
+<definition name="OSRT:OriginUpdateDelay" namespace="OSRT">
```

Comment:
There is a mismatch between name of file and content

Reply:
Good catch.